mapping.csv

```diff
@@ -392,7 +392,7 @@
 "^kraken$",kraken,
 "^fosniw$",fosniw,
 "^patriot$",patriot,
-"^koobface$",koobface,malpedia,https://en.wikipedia.org/wiki/Koobface
+"^(koobface|jnanabot)$",koobface,malpedia,https://en.wikipedia.org/wiki/Koobface
```

Comment:
Do you have a citable source for this mapping?

Reply:
Seems like this is one of a couple of names for the OSX port of Koobface:
- https://www.sophos.com/en-us/threat-center/threat-analyses/viruses-and-spyware/Troj~Boonana-A.aspx
- https://www.theregister.co.uk/2011/01/19/mac_linux_bot_vulnerabilities/
- https://www.intego.com/mac-security-blog/intego-security-memo-trojan-horse-osxkoobface-a-affects-mac-os-x-mac-koobface-variant-spreads-via-facebook-twitter-and-more/